tilecloud_chain/__init__.py

```diff
@@ -1279,4 +1279,10 @@ def _safe_generator(generator, time_message=None):
             exit("User interrupt")
         except StopIteration:
             return
+        except RuntimeError as e:
```

Comment:
Will it continue working with python < 3.7?

Reply:
Yes. The stuff is tested with python 3.5 and 3.6. It's just sad that one cannot test with 3.7 on travis... we would have cought that problem earlier...

Reply:
@pvalsecc, see https://blog.travis-ci.com/2018-11-08-xenial-release. Not sure python 3.5 is still there though.